src/pages/Category/Category.js

```diff
@@ -1,8 +1,33 @@
 import React, { Component } from 'react';
 import './Category.scss';
+import Mainimage from './Mainimage';
+import Maintext from './Maintext';
+import Filter from './Filter';
+import Product from './Product.js';
+import Productlist from './Productlist.js';
 
 export default class Category extends Component {
+  state = {
+    filter: 'name',
+  };
+
+  changeColor = e => {
+    this.setState({
+      filter: e.target.name,
+    });
+  };
+
   render() {
-    return <div>Hello, Category!</div>;
+    return (
+      <div className="category">
+        <div className="nav-bar" />
+        <Mainimage />
+        <Maintext />
+        <div className="category-product-contents">
```

Comment:
```suggestion
        <MainImage />
        <MainText />
        <div className="category-product-contents">
```
컴포넌트 명은 파스칼 케이스로 적어주세요

Reply:
네, 수정했습니다!